javascript/i18n/phonenumbers/phonenumberutil.js

```diff
@@ -1096,6 +1096,22 @@ i18n.phonenumbers.PhoneNumberUtil.normalizeDigitsOnly = function(number) {
 
 
 /**
+ * Normalizes a string of characters representing a phone number. This strips
+ * all characters which are not diallable on a mobile phone keypad (including
+ * all non-ASCII digits).
+ *
+ * @param {string} number a string of characters representing a phone number.
+ * @return {string} the normalized string version of the phone number.
+ */
+i18n.phonenumbers.PhoneNumberUtil.normalizeDiallableCharsOnly =
+    function(number) {
+
+  return i18n.phonenumbers.PhoneNumberUtil.normalizeHelper_(number,
+      i18n.phonenumbers.PhoneNumberUtil.DIALLABLE_CHAR_MAPPINGS_, true);
```

Comment:
Sure. We are a bit inconsistent since we don't do it in the next method in Java either (convertAlphaCharactersInNumber) but oh well.